monasca-python/Dockerfile

```diff
@@ -0,0 +1,5 @@
+ARG PYTHON_VERSION="2"
+FROM python:${PYTHON_VERSION}-alpine
```

Comment:
Done, added also variable that will easily allow to build different variants of the base image for the case where alpine gets bumped. That case we will only need to update ``build.yml`` I guess.